package.json

```diff
@@ -496,7 +496,7 @@
     "supertest-as-promised": "^4.0.2",
     "tape": "^4.13.0",
     "tree-kill": "^1.2.2",
-    "typescript": "3.7.2",
+    "typescript": "3.9.3",
```

Comment:
Dare we bump to 3.9.5? 
https://github.com/microsoft/TypeScript/releases/tag/v3.9.5

Reply:
will do